commands/charm/confirm/confirm.go

```diff
@@ -71,18 +88,31 @@ var promptTemplate = `
 `
 
 var resultTemplate = `
+{{- if RenderResult .FinalValue -}}
 {{- if.FinalValue -}}{{success promptPrefix}}{{else}}{{error promptPrefix}}{{end}}
```

Comment:
should there be a space here `if. FinalValue`?

Reply:
Yes but somehow it works without one 😯  https://go.dev/play/p/LSrWfMTlniJ